templates/cron-prepare.sh.j2

```diff
@@ -33,7 +33,9 @@ function title {
 # popd
 
 title "Postgres DB dump"
-/usr/bin/pg_dump --encoding=UTF8 {{ backups_role_db_name }} > {{ backups_role_tmp_path }}/pg_dump
+for DB in {{ backups_role_db_names | map('quote') | join(' ')}}; do
```

Comment:
this is not working as expected. Result is: `cats dogs` instead of the desired `"cats" "dogs"`. We can't simply insert arbitrary names into a bash script to control a loop...

Reply:
solved with ef60839